### PR TITLE
Com 41 component user data

### PR DESCRIPTION
**Pull request by Vicente96**
- Website design using styled Component.
- Inputs validations using yup library.
 -The use of API is going to implemented in the next commit.

### CODE REVIEW

File: src/components/Login/Login.css [hunk 1]
```diff
@@ -1,12 +1,13 @@
 
-.elpadrote {
+.principal {
```
Comment (gonzajo) [inline src/components/Login/Login.css]: use styled components plis
```suggestion
.container {
```

File: src/components/SignUp/SignUp.js [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+          hideProgressBar: false,
+          closeOnClick: true,
+          pauseOnHover: false,
+          draggable: true,
+          progress: undefined,
+          theme: "light"
+        });
+      }else if (validarCampos === true) {
+        alert("Datos correctos");
+        console.log(JSON.stringify(datos));
```
Comment (gonzajo) [inline src/components/SignUp/SignUp.js]: use toas instead of alert, and also remove that console.log

File: src/components/SignUp/SignUp.js [hunk 3]
```diff
@@ -0,0 +1,317 @@
...
+            height="70"
+            alt="Logotipo Empresa"
+          />
+         
+          REGISTRO
+         
+        </ContainerHeader>
+        </Link>
+        
+        <Form onSubmit={EnviarDatos}>
```
Comment (gonzajo) [inline src/components/SignUp/SignUp.js]:
```suggestion
        <Form onSubmit={handleOnSumit}>
```

**Review by gonzajo — CHANGES_REQUESTED** [review]

File: src/components/SignUp/SignUp.js [hunk 4]
```diff
@@ -0,0 +1,371 @@
...
+          datos.birth_date,
+          datos.gender,
+          datos.state,
+          datos.town,
+          datos.neighborhood,
+          datos.program,
+          datos.tags,
+          datos.emprendedor,
+          datos.aliado
+        );
```
Comment (gonzajo) [inline src/components/SignUp/SignUp.js]: why you are sending each property instead of sending the object `datos`?

**Review by gonzajo — CHANGES_REQUESTED** [review]

File: src/components/SignUp/SignUp.js (hunk 4 above)
Comment (Vicente96) [reply src/components/SignUp/SignUp.js]: Improve the code, I send an object now.

File: src/api/signup.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    state,
+    town,
+    neighborhood,
+    program,
+    tags,
+    emprendedor,
+    aliado
+  });
+};
+
```
Comment (gonzajo) [inline src/api/signup.js]:
```suggestion
export const CreateUser = async data => axiosInstance.post(`/users`,  data);

```

File: src/components/SignUp/SignUp.js [hunk 6]
```diff
@@ -0,0 +1,368 @@
...
+            datos.birth_date,
+            datos.gender,
+            datos.state,
+            datos.town,
+            datos.neighborhood,
+            datos.program,
+            datos.tags,
+            datos.emprendedor,
+            datos.aliado
+          );
```
Comment (gonzajo) [inline src/components/SignUp/SignUp.js]:
```suggestion
          const {
            data: { data: {} = {} },
          } = await CreateUser(datos);
```

**Review by gonzajo — CHANGES_REQUESTED** [review]
just minor feedback

**Review by gonzajo — APPROVED** [review]

**Pull request merged**